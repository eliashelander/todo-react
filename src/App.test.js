import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import App from './App';

describe('App', () => {
  it('Should render without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<App />, div);
    ReactDOM.unmountComponentAtNode(div);
  });
  it('Should match snapshot', () => {
    const app = renderer
      .create(<App />)
      .toJSON();
    expect(app).toMatchSnapshot();
  });
});
