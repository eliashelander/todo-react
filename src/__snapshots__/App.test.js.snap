// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Should match snapshot 1`] = `
<div
  className="todo-wrapper"
>
  <div
    className="form-wrapper"
  >
    <form
      action="post"
      className="form box-shadow"
      onSubmit={[Function]}
    >
      <label
        className="form__label"
        htmlFor="title"
      >
        Todo title
        <input
          className="form__input"
          id="title"
          name="title"
          onChange={[Function]}
          type="text"
          value=""
        />
      </label>
      <label
        className="form__label"
        htmlFor="description"
      >
        Describe your todo
        <input
          className="form__input"
          id="description"
          name="description"
          onChange={[Function]}
          type="text"
          value=""
        />
      </label>
      <button
        className="btn btn--add"
        name="add"
        type="submit"
      >
        Add
      </button>
    </form>
  </div>
  <section
    className="todo-list"
  />
</div>
`;
